pyproject.toml

```diff
@@ -41,6 +41,7 @@ packages = [
   {include = "canvas_sdk"},
   {include = "logger"},
   {include = "plugin_runner"},
+  {include = "pubsub"},
```

Comment:
I don't believe we need this, see: https://github.com/canvas-medical/canvas-plugins/issues/144#issuecomment-2430548755

Reply:
@aduane The new CLI command `run_plugins` imports run_server from plugin_runner, which in turn imports log, bringing in `Publisher` from `pubsub.pubsub`.


```
Traceback (most recent call last):
  File "/.local/bin/canvas", line 5, in <module>
    from canvas_cli.main import app
  File "/canvas_cli/main.py", line 10, in <module>
    from canvas_cli.apps.run_plugins import run_plugin, run_plugins
  File "/canvas_cli/apps/run_plugins/__init__.py", line 1, in <module>
    from canvas_cli.apps.run_plugins.run_plugins import run_plugin, run_plugins
  File "/canvas_cli/apps/run_plugins/run_plugins.py", line 1, in <module>
    from plugin_runner.plugin_runner import run_server
  File "/plugin_runner/plugin_runner.py", line 28, in <module>
    from logger import log
  File "/logger/__init__.py", line 1, in <module>
    from logger.logger import PluginLogger
  File "/logger/logger.py", line 5, in <module>
    from pubsub.pubsub import Publisher
ModuleNotFoundError: No module named 'pubsub'
```

Reply:
🤦  yep, you are correct. Thanks!